### PR TITLE
Command utils + train

### PR DESCRIPTION
**Pull request by KuuCi**
Reorganize directories to keep scripts in folder command_cli

### CODE REVIEW

**Comment by KuuCi** [comment]
test-cli-tKYJTz runs:
`composer -c llmfoundry train /mnt/config/parameters.yaml || (echo "Command failed - killing python" && pkill python && exit 1)`

image


test-cli-fC5kEk runs:
`composer -c llmfoundry train /mnt/config/parameters.yaml || (echo "Command failed - killing python" && pkill python && exit 1)`

image

File: tests/a_scripts/train/test_train_inputs.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 from omegaconf import DictConfig
 from omegaconf import OmegaConf as om
 
-from llmfoundry.train import train  # noqa: E402
+from llmfoundry.command_utils import train  # noqa: E402
```
Comment (dakinggg) [inline tests/a_scripts/train/test_train_inputs.py:12]: why noqa?

**Review by dakinggg — APPROVED** [review]

File: tests/a_scripts/train/test_train_inputs.py (hunk 1 above)
Comment (KuuCi) [reply tests/a_scripts/train/test_train_inputs.py:12]: It was always there :/

Comment (dakinggg) [reply tests/a_scripts/train/test_train_inputs.py:12]: sure, but can you remove it?

**Pull request merged**